pkg/signup/keymanager.go

```diff
@@ -0,0 +1,172 @@
+package signup
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
```

Comment:
shouldn't you use `github.com/sirupsen/logrus` instead?

Reply:
This is what was used in the initial scaffolding. I did not change something there..

Reply:
ok, but that's the logger impl we've used on the fabric8 services, and I believe we could (should?) keep using it here.

Reply:
Let's do this in a seperate PR then. This would touch more than just the key management code.